pkg/container/docker_run.go

```diff
@@ -357,6 +415,7 @@ func (cr *containerReference) create(capAdd []string, capDrop []string) common.E
 		if err != nil {
 			return errors.WithStack(err)
```

Comment:
Been playing with this branch and noticed that errors here are getting swallowed. I added:
```golang
logger.Debugf("Create Container Err ==> %+v", err)
```
And now have a "Duplicate mount point" error to chase

Reply:
Commenting out the working directory mount where service containers are spawned in `pkg/runner/run_context.go` got everything working. That doesn't seem ideal though as I assume that was needed for something

Reply:
For clarity, the line in question 
```golang
// mnt[name] = filepath.Dir(workdir)
```

Reply:
What are you trying to fix?